packages/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -361,6 +361,23 @@ class Transforms {
           }
         }
 
+        // Decrypt encrypted text map if present
+        if (activity.encryptedTextKeyValues !== undefined) {
+          for (let [value] of activity.encryptedTextKeyValues.values()) {
+            promises.push(requestWithRetries(ctx.webex.internal.encryption, ctx.webex.internal.encryption.decryptText,
+              [activity.encryptionKeyUrl, value])
+              .then((decryptedMessage) => {
+                value = decryptedMessage;
```

Comment:
I don't think this sets the decrypted data.

<img width="324" alt="image" src="https://user-images.githubusercontent.com/5473486/203596328-18d85653-3e2f-495e-972c-2c5d9b8791a3.png">

I think you'd need to set the value directly on the object. e.g.

    activity.encryptedTextKeyValues.values()[0] = decryptedMessage

But please note that `values()[0]` is pseudo code, I can't recall how this object is structured, but it should be simple to modify.

Reply:
Yes, cat is not a reference, it's a new local variable